### PR TITLE
ENH: Add make_sidecar_table macro that accepts a table name

### PR DESCRIPTION
**Pull request by effigies**
Starting to convert our in-text tables to references to our in-schema tables.

### CODE REVIEW

File: tools/mkdocs_macros_bids/macros.py [hunk 1]
```diff
@@ -196,6 +196,35 @@ def make_metadata_table(field_info, src_path=None):
     return table
 
 
+def make_sidecar_table(table_name, src_path=None):
+    """Generate a markdown table of metadata field information.
+
+    Parameters
+    ----------
+    field_names : dict
```
Comment (rwblair) [inline tools/mkdocs_macros_bids/macros.py:204]: table_name or field_names? sting or list of stings instead of dict.

Comment (effigies) [reply tools/mkdocs_macros_bids/macros.py:204]: Oh, I probably didn't update this. Thanks for pointing it out...

File: tools/schemacode/bidsschematools/render.py [hunk 2]
```diff
@@ -499,36 +514,98 @@ def make_obj_table(subschema, field_info, src_path=None, tablefmt="github"):
...
+    Returns
+    -------
+    table_str : str
+        The tabulated table as a Markdown string.
+    """
+    if isinstance(table_name, str):
+        table_name = [table_name]
+    fields = {}
+    for table in table_name:
+        fields.update(schema.rules.sidecars[table].fields)
```
Comment (rwblair) [inline tools/schemacode/bidsschematools/render.py:571]: May be worth printing on key collision, or having a rule for deciding which entry to keep.

Comment (effigies) [reply tools/schemacode/bidsschematools/render.py]: Changed it to an error.

**Comment by effigies** [comment]
TODO:

1) [qMRI appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/11-qmri.html#metadata-requirements-for-qmri-maps)
2) [Tabular files](https://bids-specification.readthedocs.io/en/stable/02-common-principles.html#tabular-files)
3) [dataset_description.json](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#dataset_descriptionjson) (and [here](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#derived-dataset-and-pipeline-description))
4) [Phenotypic and assessment data](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#phenotypic-and-assessment-data)
5) [Genetic Information](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/08-genetic-descriptor.html#genetic-information)

**Pull request merged**